prac_08/silver_service_taxi.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404/CP5632 Practical
+Silver Service Taxi class
+"""
+from prac_08.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    """Specialised version of a Taxi that includes a fanciness factor."""
+    flagfall = 4.5
+
+    def __init__(self, name, fuel, fanciness):
+        """Initialise a Silver Service Taxi ."""
+        super().__init__(name, fuel)
+        self.fanciness = fanciness
+        self.price_per_km = fanciness * Taxi.price_per_km
```

Comment:
This should be self.price_per_km. It might be the same, but if we ever changed this taxi's price, it would be different. By default, always use self here unless you know you want the class variable independent of any local changes.